onnx/reference/ops/_op.py

```diff
@@ -151,20 +151,25 @@ def __init__(self, onnx_node: NodeProto, run_params: dict[str, Any]):
     def is_axes_empty(self, axes):
         return axes is None or axes.shape == (0,)
 
-    def handle_axes(self, axes):  # noqa: PLR0911
+    def handle_axes(self, axes, noop_with_empty_axes=False):  # noqa: PLR0911
         if isinstance(axes, tuple):
-            if len(axes) == 0:
+            if len(axes) == 0 and not noop_with_empty_axes:
```

Comment:
So: I guess the numpy convention is that `axes=None` means reduce over all axes?